businessRules/ideas.js

```diff
@@ -1,9 +1,65 @@
-const { getAllIdeas } = require('../models/ideas');
+const {
+    getAllIdeas,
+    createIdea,
+    updateIdea,
+    deleteIdea,
+} = require('../models/ideas');
 
 function getIdeas() {
     return getAllIdeas();
 }
 
+function sortedByDonationSumIdeas() {
+    const ideasList = getIdeas();
+
+    const sortedIdeasList = ideasList.sort(
+        (a, b) => a.totalDonationSum - b.totalDonationSum
+    );
+    return sortedIdeasList;
+}
+
+function createIdeas(header, description, askedSum, userId) {
+    if (header === null) {
+        throw new Error('Error. Please, check your header.');
```

Comment:
`throw` terminates function execution, so you don't need to do `if ... else if...` - you can simply do `if`:
```js
if (!header) {
  throw new Error('Error. Please, check your header.');
}

if (!description) {
  throw new Error('Error. Please, check your description.');
}
```

Reply:
Updated and tested.